py2hwsw/lib/hardware/arith_logic/iob_edge_detect/iob_edge_detect.py

```diff
@@ -5,22 +5,34 @@
 
 def setup(py_params_dict):
     attributes_dict = {
-        "generate_hw": False,
+        "generate_hw": True,
+        "confs": [
+            {
+                "name": "EDGE_TYPE",
+                "descr": "Edge detection type",
+                "type": "P",
+                "val": '"rising"',
+                "min": "NA",
+                "max": "NA",
+            },
+            {
+                "name": "OUT_TYPE",
+                "descr": "Output type",
```

Comment:
```suggestion
                "descr": "Output type. Options: "step", "pulse"",
```

Reply:
update to:
```python
"descr": "Output type. Options: 'step', 'pulse'",
```
for proper python string escaping